static/src/javascripts/projects/commercial/modules/commercial-features.js

```diff
@@ -45,6 +45,9 @@ class CommercialFeatures {
             document.documentElement.classList.contains('has-sticky');
         const newRecipeDesign =
             config.page.showNewRecipeDesign && config.tests.abNewRecipeDesign;
+        const isSecureContact = config
+            .get('page.pageId', '')
+            .includes('contact-the-guardian-securely');
```

Comment:
Might be safer to do a full match?

Reply:
I was torn on how specific to be, I'd like to catch the page again if it gets reposted (where I suspect the parts of the path based on the publication date may change, but the title will not)

Maybe it would be better to check that the pageId ends with 'contact-the-guardian-securely'? I'm doing the Scala template changes now and am thinking of using `id.endsWith('contact-the-guardian-securely')` in the check

Reply:
If it does get reposted, the path can always be updated. The risk of catching the wrong page id outweighs the convenience of dealing with the fairly benign edge case of reposting. Also, for future readers, a full path makes it much easier to find out which page this logic was aimed at.

Reply:
Good points! I have another PR open for the inlined \<scripts\> so will go make both checks more specific 👍